README.md

```diff
@@ -1337,9 +1337,9 @@ The Lawyer can win in multiple ways:
 
 ## Pursuer
 ### **Team: Neutral**
-The Pursuer is still a neutral role, but has a different goal to win the game: Them has to be alive, when the game ends (no matter who caused the win).
-In order to achieve this goal, the Pursuer has the ability "Blank": Them can fill a killers (this also includes the Sheriff) weapon with a blank, so if the killer attempts to kill someone, the killer will miss the target and all cooldowns will be triggered as usual.
-If the killer blanks, no shields (e.g. Medic shield or Time Master shield) will be triggered.
+The Pursuer is still a neutral role, but has a different goal to win the game; they have to be alive when the game ends (no matter who causes the win).
+In order to achieve this goal, the Pursuer has an ability called "Blank", where they can fill a killers (this also includes the Sheriff) weapon with a blank. So, if the killer attempts to kill someone, the killer will miss their target, and their cooldowns will be triggered as usual.
```

Comment:
```suggestion
In order to achieve this goal, the Pursuer has an ability called "Blank". Using Blank on a player with any role that can kill, including the Sheriff, fills that killer's weapon with a blank. If the killer tries to kill someone, the killer will miss their target, and their cooldowns will be triggered as usual.
```
`killers` -> `killer's`, but I'd also like to suggest a bit more rewording here; I find the parenthetical makes it a little bit hard to keep track of who's being talked about.

Ambiguity that would be nice to answer here: what happens if Blank is used on a non-killer? Does it appear to work and silently do nothing? Does it fail? Is it unavailable in the first place?

Reply:
True, seems to be a good idea!